unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDView.cs

```diff
@@ -281,4 +300,8 @@ public void CleanAllEntries()
 
         dateSeparators.Clear();
     }
+    
+    public void SetGotoPanelStatus(bool isActive) {
```

Comment:
```suggestion
    public void SetGotoPanelStatus(bool isActive)
    {
```

Reply:
Also, I see `gotoPanelVisible` is being modified in `ChatEntry` why do we need this method for?